docs/rules/no-duplicate-literal-elements.md

```diff
@@ -0,0 +1,149 @@
+# Disallow duplicate literal elements in `Array`, `Set` or `Map` key
+
+✅ *This rule is part of the [recommended](https://github.com/sindresorhus/eslint-plugin-unicorn#recommended-config) config.*
+
+It is hard to know if there is duplicate in hardcoded value when `Array`, `Set`, or `Map` key grows larger.
```

Comment:
This description seems vague to me. Sets and maps make sense for this rule since they do not allow duplicate elements or keys respectively, so this description should mention that. What would be the reason for arrays?

Reply:
Actually, I’m not sure why I need to check the arrays...

There are many cases where the contents of the array are duplicated.
like this: `((x, y) => Math.pow(x, y)).apply(null, [2, 2])`

I will confirm how to rewrite the description, thx~